endpoints/v2/__init__.py

```diff
@@ -58,6 +60,15 @@ def handle_readonly(ex):
     return response
 
 
+@v2_bp.app_errorhandler(UpstreamRegistryError)
+def handle_readonly(error):
+    e = InvalidRequest(message=str(error))
```

Comment:
`InvalidRequest` returns an error with `INVALID_REQUEST` code. Although that error code is not part of the distribution spec, it's used in quite a few places throughout quay's v2 api.
we can definitely use another error here, I'm just not sure which one.
suggestions are most welcome.